components/src/core/UserMenu/UserMenu.tsx

```diff
@@ -200,7 +212,18 @@ const UserMenuRender: React.ForwardRefRenderFunction<unknown, UserMenuProps> = (
                 ...menu.props,
             });
         }
-        return (
+        return useMediaQuery(`(max-width:${useBottomSheetAt}px)`) && !disableBottomSheet ? (
+            <Drawer
+                data-cy="bottom-sheet"
+                anchor={'bottom'}
+                transitionDuration={250}
+                open={Boolean(anchorEl)}
+                onClose={closeMenu}
+                classes={{ paper: clsx(defaultClasses.bottomSheet, classes.bottomSheet) }}
+            >
```

Comment:
Should we extend the `{...MenuProps}` here too?

Reply:
Or to simplify things a little bit, have some BottomsheetProp of the type DrawerProp

Reply:
Yeah we can do that...I would prefer the second option (`BottomsheetProps`) since the types for Menu and Drawer are not compatible.

I'm not sure if people are really going to need to change much with the drawer (unlike with the menu where they are likely to want to control it's position), but it doesn't hurt to give more customizability We've certainly given people enough options to do crazy things in the other components 😆